.github/workflows/pull_request.yml

```diff
@@ -135,7 +135,7 @@ jobs:
       data: ${{ needs.RunConfig.outputs.data }}
   # stage for jobs that do not prohibit merge
   Tests_3:
-    needs: [RunConfig, Tests_1, Tests_2]
+    needs: [RunConfig, Builds_1, Tests_1, Builds_2, Tests_2]
```

Comment:
Can you add a comment here and in master why they're different? It's unclear, and eventually we'll break it